OpenRA.Mods.Common/Traits/Buildings/Building.cs

```diff
@@ -180,12 +181,17 @@ public Building(ActorInitializer init, BuildingInfo info)
 			occupiedCells = FootprintUtils.UnpathableTiles(self.Info.Name, Info, TopLeft)
 				.Select(c => Pair.New(c, SubCell.FullCell)).ToArray();
 
+			targetableCells = FootprintUtils.UnpathableTiles(self.Info.Name, Info, TopLeft)
```

Comment:
Note to reviewers: This duplication is intended for now, #13553 will update this part as soon as this PR is merged.
This is primarily a measure to keep the diff of the next revision of #13553 in check, as code-wise that will be harder to review than this PR.